package-lock.json

```diff
@@ -127,181 +127,290 @@
       }
     },
     "@google-cloud/common": {
-      "version": "0.16.2",
-      "resolved": "https://registry.npmjs.org/@google-cloud/common/-/common-0.16.2.tgz",
-      "integrity": "sha512-GrkaFoj0/oO36pNs4yLmaYhTujuA3i21FdQik99Fd/APix1uhf01VlpJY4lAteTDFLRNkRx6ydEh7OVvmeUHng==",
-      "requires": {
-        "array-uniq": "^1.0.3",
+      "version": "0.23.0",
```

Comment:
That's a lot of dependency changes (risk) to bundle with this PR. Are they needed?

Reply:
We've been accepting updates from greenkeeper, I just don't think it updated the package-lock.